### PR TITLE
Add Golint and fix all issues

### PR DESCRIPTION
**Pull request by philipithomas**
Go Report Card reported 26 linting issues with Buffalo:

https://goreportcard.com/report/github.com/gobuffalo/buffalo

I went through and corrected all of them, plus added linting to the Travis test. 

---

@markbates As discussed, I just made standard golint pass. You may want to keep grifts imports as a top-level rather than namespaced. (I personally found this confusing because of how many different type of "Context" types get passed around)

So - if you want me to do some modifications to allow for dot imports and revert those files, let me know.

### CODE REVIEW

File: examples/hello-world/grifts/routes.go [hunk 1]
```diff
@@ -4,11 +4,11 @@ import (
 	"os"
 
 	"github.com/gobuffalo/buffalo/examples/hello-world/actions"
-	. "github.com/markbates/grift/grift"
+	mbgrift "github.com/markbates/grift/grift"
 	"github.com/olekukonko/tablewriter"
```
Comment (markbates) [inline examples/hello-world/grifts/routes.go]: Why are we using a special name here?

**Comment by markbates** [comment]
Other than the one question I have, this looks good.

File: examples/hello-world/grifts/routes.go (hunk 1 above)
Comment (philipithomas) [inline examples/hello-world/grifts/routes.go]: Whoops, just pushed a commit to fix it. 

when I first started this PR . . . I misread something and thought there was a namespace collision. But there's not.

**Comment by philipithomas** [comment]
@markbates Corrected

**Comment by markbates** [comment]
Thanks!

**Pull request merged**